### DISCUSSION TITLE
Introduce DKs

### DISCUSSION THREAD

**Discussion by Erson33** [chromiecraft/chromiecraft#6056] [General]
Hello! First off, love you guys and all the work you're doing on CC.

I know the present plan is for DKs to come out with the Northrend phase but I was wondering if the team was considering moving this up. I think there are quite a few potential benefits.

1. Current queues for RDF at 70 are more than an hour, typically waiting on tanks. Introducing another tank class would lead to more runs and thereby more testing.
2. Would get people leveling through the Outland zones/dungeons again.
3. Potential to uncover bugs/issues with DKs within current content (T4 raids) that would otherwise not be known.
4. More time to uncover and address DK issues in general.

The main issue people share is that there are no DK tier sets. Fortunately, a majority of the tier bonuses in TBC are not that impactful with WOTLK specs. I think the non-tier gear options would be sufficient. DK PVP sets would need to be created but I imagine they could be largely copied from the warrior pieces and the set bonuses remain largely static and thus could be copied from WOTLK PVP sets

Overall, I think introducing the DK class early would have a lot of benefits and could be a good way to keep server population high between Kara and T5.

Thanks for the consideration!

**Comment by balleny** [comment]
> The main issue people share is that there are no DK tier sets

the main issue people raise is that the class wasnt balanced at all for level 70 content and would introduce a lot more issues right now. Further, this would also (again) break progression.

Lastly, the topic has been raised quite a few times. If you want to bring further arguments, i would recommend to go through the previous suggestions first, collect the statements and then see if you can bring additional arguments that could actually influence the current position

**Reply by Never6** [reply]
um, a lot of stuff isnt balanced for level 70 content so using that as your main argument is just silly

**Reply by Never6** [reply]
DSac
Fury war as a spec
Feral cat as a spec
Pretty much every healer because they lack haste 
Arms war right now is legit S tier dps but I would be curious to know how many arms warriors you have ran into in ICC, because I bet i can guess the numbers (its 0)
Using the argument "its not balanced around 70 content" is probably the dumbest argument that you could make
yeah its not balanced NOTHING IS we play on 3.3.5 patch everything is balanced around T10

**Reply by Never6** [reply]
If you want more examples on things that aren't not balanced around playing 70 content on 3.3.5 patch I would be happy to give them to you but I think i made my point very clear.
And i have tested DKS on ptr the misconception that they would absolutely destroy the meta is just flat out ridiculous 
they were good especially with BS maces, but specs like Affliction Arcane Arms Assa (when bosses arent poison immune) Ret still exist and are just as good if not better
There is no good reason to not release them right now

**Comment by Never6** [comment]
maybe one day...
